README.md

```diff
@@ -70,12 +70,20 @@ Note that this will pull the Docker image and may take a few minutes.
 docker compose run base
 ```
 
+This will start the container and drop you into a bash shell.
+
 Once you're in the container, check that you can launch a simulated world and the PyRoboSim UI appears on your screen.
 
 ```bash
 ros2 run delib_ws_worlds run
 ```
 
+You can attach to the running container from a different console, using the command:
+
+```bash
+docker compose run base bash
```

Comment:
Did this work for you, or did you need to add `run -it base bash` for interactive / tty? (That's the usual way to attach)

Reply:
`docker compose run base` seems to work for me.  Are we defaulting to `-it` in the entry point?

Simpler is better for our workshop commands.

Reply:
it worked without the `-it`